config/website.js

```diff
@@ -2,9 +2,9 @@
 
 module.exports = {
   pathPrefix: '/', // Prefix for all links. If you deploy your site to example.com/blog your pathPrefix should be "blog"
-  siteTitle: 'HackNITR 3.0', // Navigation and Site Title
+  siteTitle: 'HackNITR 4.0', // Navigation and Site Title
   siteTitleAlt: 'Official Website of HackNITR', // Alternative Site title for SEO
-  siteTitleShort: 'HackNITR 3.0', // short_name for manifest
+  siteTitleShort: 'HackNITR 4.0', // short_name for manifest
   siteUrl: 'https://hacknitr.tech', // Domain of your site. No trailing slash!
```

Comment:
.tech ko .com

Reply:
Also the site description

Reply:
Review this [page](https://github.com/dscnitrourkela/project-oregano/blob/4065cd8f390067665debdb25187b8877561f5daa/config/website.js) to check if any old links are there